pkg/agent/cniserver/ipam/hostlocal/lock.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hostlocal
+
+import (
+	"os"
+	"path"
+
+	"github.com/alexflint/go-filemutex"
+)
+
+// This code was copied from https://github.com/containernetworking/plugins/blob/v1.3.0/plugins/ipam/host-local/backend/disk/lock.go
```

Comment:
could we import `github.com/containernetworking/plugins/plugins/ipam/host-local/backend` given the package is already a dependency? https://github.com/antrea-io/antrea/blob/4a52bcafe3b56b39fccda2cd9016f66db166e65c/go.mod#L22

Reply:
Yes, we actually use it pretty extensively. Changed to an import.